addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -89,6 +89,13 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         const themeBodyLineHeightEl = $html.find(
             "[data-selector='theme-paragraph'] we-input[data-customize-website-variable][data-variable='body-line-height']"
         )[0];
+        // Remove the input-border-width-sm and input-border-width-lg from the input-border-width
+        ['input-border-width-sm', 'input-border-width-lg'].forEach(variable => {
+            const element = $html.find(
+                `[data-selector='theme-input'] we-input[data-customize-website-variable][data-variable='${variable}']`
+            )[0];
+            element.remove();
+        });
```

Comment:
Why separate the definition of `themeBodyLineHeightEl` from its use ?
For readability, could you put this block at the end of the method instead ? This way it will follow the same order as the content of the panel that is modified.

Also, do we agree that this is a fix for stable that will be removed in the master forward-port ?

Reply:
@bso-odoo My bad! I had no intention to separate `themeBodyLineHeightEl` from its use, initially i used a variables to keep those data in array and then find the selector and remove them, with recent change made it look like we have separated the code, absolutely it should be kept at the end of the function for readability , i will make those changes ASAP!

> Also, do we agree that this is a fix for stable that will be removed in the master forward-port ?

Yes the changes made on this PR is for stable versions only, and this code will be removed in the master forward-port, along with the changes in the xml to removes `input-border-width-sm`, `input-border-width-lg`.
Thanks!